src/core/shutdown.c

```diff
@@ -323,6 +323,9 @@ int main(int argc, char *argv[]) {
         if (!in_container)
                 sync_with_progress();
 
+        /* Prevent coredumps */
```

Comment:
Is there a point to this comment? One can tell that you want to prevent coredumps from the name of the function.

Reply:
You're absolutely right. At least it's consistent. I'll take more care next time...